gvfs-helper-client.c

```diff
@@ -231,29 +224,33 @@ static int ghc__get__receive_response(struct child_process *process,
 	return err;
 }
 
+/*
+ * Select the preferred ODB for fetching missing objects.
+ * This should be the alternate with the same directory
+ * name as set in `gvfs.sharedCache`.
+ *
+ * Fallback to .git/objects if necessary.
+ */
 static void ghc__choose_odb(void)
 {
 	struct object_directory *odb;
 
 	if (ghs__chosen_odb)
 		return;
 
+	ghs__chosen_odb = the_repository->objects->odb;
```

Comment:
Just adding a reminder to maybe rename refactor the `ghc` and `ghs`, as discussed offline. Those acronyms aren't very intuitive to someone reading this code for the first time. Thanks!

Reply:
I've included a few commits to rename some of my cryptic symbols.  Hopefully that will help.

Reply:
GHC clearly stands for Gigantic Hadron Collider.